lib/quake_parser.ex

```diff
@@ -41,13 +41,37 @@ defmodule QuakeParser do
       ]
   ```
   """
-  @spec start(String.t()) :: list(__MODULE__.t())
-  def start(path) do
+  @spec parse(String.t()) :: list(__MODULE__.t())
+  def parse(path) do
     File.stream!(path)
     |> find_games
     |> Enum.map(&parse_game/1)
   end
 
+  @doc """
+  Build a map with the kills of each player for all games of the log.
+
+  ## Examples
+  ```
+      iex(4)> QuakeParser.scoreboard("test/fixtures/log.txt")
+      %{
+        "Assasinu Credi" => 22,
+        "Chessus" => 0,
+        "Dono da Bola" => 12,
+        "Isgalamido" => 16,
+        "Mal" => -3,
+        "Oootsimo" => 20,
+        "Zeh" => 9
+      }
+  ```
+  """
+  @spec scoreboard(String.t()) :: %{String.t() => Integer.t()}
+  def scoreboard(path) do
+    parse(path)
+    |> Enum.map(fn %__MODULE__{kills: kills} -> kills end)
+    |> build_full_scoreboard
```

Comment:
```suggestion
    |> build_full_scoreboard()
```

Se vc rodar o credo ele vai reclamar por não chamar com ()

BTW não dava pra ter feito isso com reduce? geralmente fica mais fácil de entender do que usar recursão

Reply:
Provavelmente daria sim, mas aqui eu quis aplicar um caso de recursão que a gente ta vendo no grupo de estudos :D